roles/letsencrypt/tasks/dns-challenge-pebble.yml

```diff
@@ -0,0 +1,40 @@
+---
+### include/role 3 - validate challenge
+- name: validate challenge only if it is created or changed
+  when: challenge is changed
+  block:
+    - name: add a new TXT record to the SAN domains
+      uri:
+        url: "http://localhost:8055/set-txt"
+        method: POST
+        body_format: json
+        # the dot after the hostname is required
+        # crtl-f "Note that a period character is required at the end of the host name here." here:
+        # https://github.com/letsencrypt/pebble/blob/master/cmd/pebble-challtestsrv/README.md
+        body:
+          {
+            "host": "_acme-challenge.{{ item }}.",
```

Comment:
Maybe the pebble server handles the requests a little bit different but I think we maybe have to use a filter for the looped items to ensure that wildcard certificates can be correctly issued.
(https://github.com/T-Systems-MMS/ansible-collection-letsencrypt/pull/21)

But i also see that it could also not be needed as we just test that the steps for the creation are working 🤔